### ISSUE TITLE
Direct links to topic/status

### ISSUE THREAD

**Issue by codification** [agical/friendsunhosted#23]
I think it would be great to be able to link directly to a status/thread/topic.

Something like http://friendsunhosted.com/?usr=someone@storage.org&update=13db452f36e3456a35f6

**Comment by jancborchardt** [comment]
What would be cool is if the direct link wouldn’t rip the update out of a person’s stream like Twitter does, but rather go to the user page, scrolling to and highlighting that status (maybe also a bit whitespace around the update).

Also, the link should be: friendsunhosted.com/#user@storage.com/first-words-of-update
(We have to use hashes, because ? parameters are sent to the server, whereas anchors are not. Also, we should have the addresses use as much natural language as possible.)

**Comment by brolund** [comment]
Possibly instead

/#myfriends/username@storage.com/status/<timestamp or text>

That wouldn't make us have to write hairy conditionals about the parts of the local path, if(emailRegex.match(firstPart)) {...}.

In the same spirit:

/#myfriends

becomes

/#myfriends/user@store.com

to see a specific friend, or

/#status 

becomes

/#status/<timestamp or text> 

to lead to a specific conversation for the local user.

Or something more nifty. :-)

**Comment by jancborchardt** [comment]
Yeah, having a bit of a namespace for that is probably good. /#user/dude@storage.com is good.

**Comment by jancborchardt** [comment]
There could even be a way to embed the children of a post on a website. That way you could embed it to have commenting on a fully static page, like Disqus.
